config/routes.js

```diff
@@ -72,6 +72,12 @@ module.exports = function(app, passport, auth) {
     //Finish with setting up the userId param
     app.param('userId', users.user);
 
+    // Game routes
+    var game = require('../app/controllers/game');
+    app.get('/api/games', game.all);
+    app.get('/api/games/:id/start', users.authToken, game.start);
```

Comment:
Wasn't able to test this on deployment. Please include screenshot on deployment.